packages/examples/contracts/coingecko-testable/Requester.sol

```diff
@@ -0,0 +1,44 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+import "@api3/protocol/contracts/rrp/requesters/RrpRequester.sol";
+
+// An example requester which expects the response from Airnode is a single int256 value.
+contract Requester is RrpRequester {
+    mapping(bytes32 => bool) public incomingFulfillments;
+    mapping(bytes32 => int256) public fulfilledData;
+
+    constructor(address airnodeAddress) RrpRequester(airnodeAddress) {}
+
+    function makeRequest(
+        address airnode,
+        bytes32 endpointId,
+        address sponsor,
+        address sponsorWallet,
+        bytes calldata parameters
+    ) external {
+        bytes32 requestId = airnodeRrp.makeFullRequest(
+            airnode,
+            endpointId,
+            sponsor,
+            sponsorWallet,
+            address(this),
+            this.fulfill.selector,
+            parameters
+        );
+        incomingFulfillments[requestId] = true;
+    }
+
+    function fulfill(
+        bytes32 requestId,
+        uint256 statusCode,
```

Comment:
Please keep an eye on [error-message](https://github.com/api3dao/airnode/pull/577) branch soon to be merged which will remove the status code param.

Reply:
Should I wait with merging this until the error message branch is merged?

Reply:
We wouldn't want to hold you back but we don't expect more updates to the `AirnodeRrp` protocol contract after we merge that branch. Our [board](https://api3dao.atlassian.net/jira/software/projects/A3P/boards/20) looks pretty clean for v0.1.0

Reply:
@acenolaza I am merging this, so please update the examples once you merge your PR :)